datacapturing/src/main/java/de/cyface/datacapturing/DataCapturingService.java

```diff
@@ -118,7 +119,7 @@
     /**
      * A facade object providing access to the data stored by this <code>DataCapturingService</code>.
      */
-    protected final PersistenceLayer<CapturingPersistenceBehaviour> persistenceLayer;
```

Comment:
I did not find any occurrences where this attribute is accessed directly from outside the package, I also checked the documentation to make sure integrators are not effected by this theoretical API change. For this reason I did not bump the API version of this file.

Reply:
Can't find the reason right now why this was necessary. I change it back for now.